src/Microsoft.Azure.SignalR.Common/ServiceConnections/MultiEndpointServiceConnectionContainer.cs

```diff
@@ -33,23 +39,21 @@ internal class MultiEndpointServiceConnectionContainer : IServiceConnectionConta
             if (generator == null)
             {
                 throw new ArgumentNullException(nameof(generator));
-            }
-
+            }
+
+            _router = router ?? throw new ArgumentNullException(nameof(router));
```

Comment:
when _endpoints is 1, it is ok to not have a `_router`?

Reply:
In current logic, DefaultRouter will be added anyway.